.claude/scripts/create_tc_hybrid.py

```
@@ -0,0 +1,189 @@
+        print("      ❌ Validation failed:", file=sys.stderr)
+        print(validate_result.stderr, file=sys.stderr)
+        sys.exit(1)
+
+    print("      ✓ Validation passed", file=sys.stderr)
+    print()
+
+    # Step 5: Create (or dry-run)
+    if args.dry_run:
+        print("[5/5] DRY RUN - Showing summary...", file=sys.stderr)
+        print()
+        subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)